plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/ui/ServerFlagSelectionDialog.java

```diff
@@ -78,28 +78,32 @@ protected Control createDialogArea(Composite parent) {
 
       @Override
       public IStatus validate(Object[] selection) {
-        if (selection.length == 1) {
+        if (selection == null || selection.length == 0) {
+          return new Status(OK, Activator.PLUGIN_ID, "");
```

Comment:
OK Statuses are normally not reported anywhere.  The plugin id is only used in the `!ENTRY` section when reporting to the Eclipse log:

```
!ENTRY org.eclipse.equinox.p2.core 4 0 2016-04-21 11:32:26.091
!MESSAGE Provisioning exception
!STACK 1
…
```